docs/pages/web/combobox.js

```diff
@@ -246,273 +110,23 @@ function ComboBoxExample(props) {
             cardSize="lg"
             title="Uncontrolled ComboBox"
             description={`An uncontrolled ComboBox should be used for basic cases where no default value or tags are required. Don't pass \`inputValue\` or \`selectedOptions\` props to keep the component uncontrolled. By passing \`inputValue\` to ComboBox, the component fully manages its internal state: any value different from \`null\` and \`undefined\` makes Combobox controlled.`}
-            defaultCode={`
-function ComboBoxExample(props) {
-  const PRONOUNS = [
-    'ey / em',
-    'he / him',
-    'ne / nem',
-    'she / her',
-    'they / them',
-    've / ver',
-    'xe / xem',
-    'xie / xem',
-    'zie / zem',
-  ];
-
-  const options = PRONOUNS.map((pronoun, index) => ({ label: pronoun, value: 'value'+index }));
-
-  const [errorMessage, setErrorMessage] = React.useState();
-
-  const handleOnBlur = ({ value }) => {
-    if (value !== "" && !PRONOUNS.includes(value)) setErrorMessage('Please, select a valid option');
-  };
-
-  const resetErrorMessage = () => (errorMessage ? setErrorMessage() : () => {});
-
-  return (
-    <Box width={400}>
-      <ComboBox
-        accessibilityClearButtonLabel="Clear the current value"
-        errorMessage={errorMessage}
-        helperText="Choose your pronouns to appear on your profile so others know how to refer to you. You can edit or remove these any time."
-        id="uncontrolled"
-        label="Pronouns"
-        noResultText="No results for your selection"
-        onBlur={handleOnBlur}
-        onChange={resetErrorMessage}
-        onClear={resetErrorMessage}
-        options={options}
-        placeholder="Add your pronouns"
-      />
-    </Box>
-  );
-}
-`}
+            sandpackExample={
+              <SandpackExample code={uncontrolled} name="uncontrolled" previewHeight={320} />
```

Comment:
nit: `name` is user-facing (it populates the accessibility label on an icon button) so we should use Sentence case

Reply:
DONE